### DISCUSSION TITLE
Let users know success/failure report counts

### DISCUSSION THREAD

**Discussion by nre** [fwupd/fwupd#7349] [Ideas]
[Vendors can see the complete reports.](https://lvfs.readthedocs.io/en/latest/telemetry.html) Could users see just the success/failure counts (on the website and/or `get-updates`)? Are there concerns about privacy or abuse? Would no one install an update with zero reports? (I would wait a few weeks but I think most people wouldn't.)

**Comment by hughsie** [comment]
There would be privacy concerns. What we do allow for security companies is to view two variables, `success` and `success_confidence` -- e.g. https://gitlab.com/fwupd/lvfs-website/-/blob/master/lvfs/firmware/models.py?ref_type=heads#L526 -- maybe it makes sense to export that on the HTML pages? I'm not sure it makes sense to be in the XML metadata.

**Reply by nre** [reply]
Thank you. `success` and `success_confidence` on the website would help me.

**Reply by nre** [reply]
I would benefit from this proposed feature now. [New ME firmware](https://fwupd.org/lvfs/devices/com.lenovo.ThinkPadR1.firmware) is available for my laptop: Would I be the first to attempt it? Or is it already proven safe by successful installs? It would be nice to have data instead of guessing.

**Reply by hughsie** [reply]
This is what I've implemented -- deploying now:

![Screenshot 2024-07-25 at 10-37-58 LVFS ColorHug2](https://github.com/user-attachments/assets/bdf3b42a-1da0-415e-84ae-1ba58af454f5)

**Comment by hughsie** [comment]
Seems to work https://fwupd.org/lvfs/devices/com.dell.uefice945437.firmware -- some device pages will take ~1h to time out of the cache and be regenerated.
